gix-upload-pack/Cargo.toml

```
@@ -0,0 +1,69 @@
+[package]
+name = "gix-upload-pack"
+version = "0.1.0"
+edition = "2021"
+authors = ["Sebastian Thiel <sebastian.thiel@icloud.com>"]
+license = "MIT OR Apache-2.0"
+description = "Git upload-pack server implementation for gitoxide"
+repository = "https://github.com/GitoxideLabs/gitoxide"
+keywords = ["git", "server", "upload-pack", "protocol"]
+categories = ["development-tools", "network-programming"]
```

Maybe for now this should be kept out of `crates.io`.

```suggestion
categories = ["development-tools", "network-programming"]
publish = false
```